python/lsst/verify/bin/inspectjob.py

```diff
@@ -110,6 +111,27 @@ def inspect_job(job):
             print("%*s = %10s" % (max_metric_length, metric, pretty_quantity))
 
 
+def build_argparser():
+    """Construct an argument parser for the ``inspect_job.py`` script.
+
+    Returns
+    -------
+    argparser : `argparse.ArgumentParser`
+        The argument parser that defines the ``inspect_job.py`` command-line
+        interface.
+    """
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        epilog='More information is available at https://pipelines.lsst.io.')
+    parser.add_argument(
+        'json_paths',
+        nargs='+',
+        metavar='json',
+        help='Verification job JSON file, or files.')
```

Comment:
I like the wording used in the comment block of this file:
"lsst.verify JSON file, or files"